src/help.go

```diff
@@ -198,6 +198,7 @@ const (
 	DescDecryptionPassword           = "decryption password"
 	DescWithLink                     = "turn off file indexing so that only those with the link can view it"
 	DescAllowDesktopLinks            = "allows docs + sheets to be pulled as .desktop files or URL linked files"
+	DescCopyLinks                    = "copy link to sources into destination folder instead of moving sources"
```

Comment:
DescKeepParent = "ensures that when moving a file into a destination, that we also retain its original parent so that it will exist in more than one folder"